### PR TITLE
Support installing plugins from a role variable

### PR DESCRIPTION
**Pull request by dougszumski**
The role variable monasca_agent_custom_plugin_repos, should contain
a list of git repos holding plugins.

Each repo should be configured to install the plugins to the correct
location on the host. See https://github.com/openstack/monasca-agent/
blob/master/docs/Customizations.md#adding-custom-check-plugins

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ virtualenv must be installed on the system.
 - monasca_user_domain_name: Keystone project domain name for Monasca
 - monasca_user_domain_id: Keystone project domain ID for Monasca
 - monasca_user: User which runs the Monasca Agent. Defaults to 'mon-agent'.
+- monasca_agent_custom_plugin_repos: A list of git repos containing custom plugins.
```
Comment (markgoddard) [inline README.md]: Could we specify a branch here?

Comment (dougszumski) [reply README.md]: That's possible. For example:

```
monasca_agent_custom_plugin_repos:
      - "https://github.com/stackhpc/stackhpc-monasca-agent-plugins.git@feature/add_ib_detection_plugin" 
```

Comment (markgoddard) [reply README.md]: Ok, perhaps a comment to specify what the format is?

**Comment by dougszumski** [comment]
* Rebase and correct author before merge.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**